### ISSUE TITLE
Compile error when integrating with existing iOS app - RCTProfile.m

### ISSUE THREAD

**Issue by zachlucas** [facebook/react-native#13988]
### Description

I'm attempting to integrate React Native into an existing iOS project, using the steps here:  http://facebook.github.io/react-native/docs/integration-with-existing-apps.html.  I'm using CocoaPods for dependencies, with React Native and Yarn `0.44.0`.  I get the following error when I try a clean compile:
```
/ios/node_modules/react-native/React/Profiler/RCTProfile.m:775:34: 'initWithTitle:message:delegate:cancelButtonTitle:otherButtonTitles:' is unavailable: not available on iOS (App Extension) - Use UIAlertController instead.
```

I have a rich notification target in my existing project, could that be the issue?  Has anyone else run into issues when using RN with app extensions?

### Reproduction Steps and Sample Code

The relevant part of my podfile:
```
    pod 'React', :path => '../node_modules/react-native', :subspecs => [
    'Core',
    'RCTText',
    'RCTNetwork',
    'RCTWebSocket', # needed for debugging
    # Add any other subspecs you want to use in your project
  ]
  # Explicitly include Yoga if you are using RN >= 0.42.0
  pod "Yoga", :path => "../node_modules/react-native/ReactCommon/yoga"
```

### Solution

React Native should compile with App Extensions

### Additional Information

* React Native version: 0.44.0
* Platform: iOS
* Development Operating System: macOS Sierra
* Dev tools: Xcode 8.3.2

**Comment by zachlucas** [comment]
After removing my rich notification target, the project compiles and runs fine.  Anyone else use React Native with an app extension target?

**Comment by zachlucas** [comment]
This seems to be fixed with `0.45.1`
